src/locales/ja-JP.ts

```
@@ -0,0 +1,144 @@
+    second: '秒',
+  },
+  address: {
+    selectRegion: '住所を選択してください',
+    deliveryTo: '配送先',
+    chooseAnotherAddress: '別の住所を選択',
+    hotCity: '人気都市',
+    selectProvince: '都道府県/地域を選択',
+  },
+  signature: {
+    reSign: '再署名',
+    unsupported: '申し訳ありませんが、現在のブラウザはCanvasをサポートしていません。このコントロールは使用できません！',
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)